library/source/datadogSdk.bs

```diff
@@ -98,6 +100,43 @@ function sdkVersion() as string
     return "1.0.0-dev"
 end function
 
+' ----------------------------------------------------------------
+' @return (string) the service name of the host channel
+' (read from the manifest)
+' ----------------------------------------------------------------
+function channelServiceName() as string
+    title = LCase(CreateObject("roAppInfo").GetTitle()) ' use lowercase version of title
+    titleLen = title.Len()
+    serviceName = ""
+
+    for i = 1 to (titleLen)
+        c = Asc(Mid(title, i, 1))
```

Comment:
should we create our own `CharAt` method? :) each time it puzzles me a bit when I see `Mid` call to get a particular char.

Reply:
We might do that indeed. So far we don't need it that much but I'll keep it in mind next time I feel like I need it.